### PR TITLE
Add in new object storage controls on encryption for impact and replication to untrusted destinations

### PR DESCRIPTION
**Pull request by mlysaght2017**
Need to work on tests, but would like to get to initial feedback on control objectives first

### CODE REVIEW

File: services/storage/object/controls.md [hunk 1]
```diff
@@ -181,3 +183,47 @@ The following validations must be performed against corresponding Control Implem
...
+- ISO/IEC 27001:2013 A.12.3.1
+- NIST SP 800-53: CP-6, CP-9
+
+### Testing Requirements
+
+The following validations must be performed against corresponding Control Implementation capabilities to ensure the Control Objective is thoroughly assessed:
+
+1. [**CCC.OS.C8.TR01**]{#CCC.OS.C8.TR.01}: Verify that access policies for cloud storage buckets and objects prevent requests with untrusted KMS keys.
+
+## CCC.OS.C9: Prevent Replication to Untrusted Destinations
```
[the remark this answers is not in the document]
Comment (mlysaght2017) [reply services/storage/object/controls.md:209]: Thinking here is that untrusted destinations (resources) are any that exist outside of a defined identity/network perimeter (e.g.) at the organization level. E.g. if targeted destination/resource is not within a member project/account/subscription within the org, then deny.

[the remark this answers is not in the document]
Comment (nas-hub) [reply services/storage/object/controls.md:209]: Got it, we may need to have a glossary of terms we use in CCC and define its scope.  Or probably just qualify the untrusted destination further contracting it with Trusted Identity Perimeter, Truster Network Perimeter similar to the term Trusted Computing Base defined in NIST/ISC2. 

The control and testing requirement LGTM

**Review by eddie-knight — CHANGES_REQUESTED** [review]
[will add note later]

**Comment by mlysaght2017** [comment]
@damienjburks @eddie-knight can you please review/approve if you haven't already - I've just added in an extra line of detail on what we mean by untrusted resource/untrusted perimeters based on feedback from @nas-hub

**Review by damienjburks — APPROVED** [review]
LGTM! 🚀

**Review by eddie-knight — APPROVED** [review]

**Pull request merged**